app/styles/components/extras.scss

```diff
@@ -14,3 +14,7 @@ a.link-item {
 .leaflet-container {
   height: 300px;
 }
+
+div#ember126 {
```

Comment:
I just merged a PR and this changed to ember234. Tomorrow it might be 132, the next day it'll be 342. We don't want to merge a PR to fix this every day. We want a solution which works once and for all

Reply:
i will fixed it